### PR TITLE
Remove mention of feature flag from Enhanced Email Macros

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Removed mention of feature flag in the Enhanced Email Macros section. Update contact support blurb.
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md [hunk 1]
```diff
@@ -29,23 +29,22 @@ Email templates use common and unique [Expression Language (EL) variables](https
...
 
 ##### What this EA feature doesn't enable
 
-Okta will not perform any automated migration of previously customized email templates for EA. This means that when you enable the feature flag to access the functionality, all email templates will revert to defaults.
-
-However, when you disable the feature flag, your old customized templates will return. If you choose to disable the feature flag, you will lose access to the customizations done while the EA was enabled.
-
-##### EA feature requirements
+Okta doesn't perform any automated migration of previously customized email templates for this EA feature. This means that when you enable the Enhanced Email Macros feature to access the functionality, all email templates will revert to defaults.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md:43]:
```suggestion
Okta doesn't perform any automated migration of previously customized email templates for this EA feature. This means that when you enable the Enhanced Email Macros feature to access the functionality, all email templates revert to defaults.
```

File: packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md [hunk 2]
```diff
@@ -29,23 +29,22 @@ Email templates use common and unique [Expression Language (EL) variables](https
...
 
-To access this self-serviced EA feature, you need to enable `ENHANCED_EMAIL_MACROS`in the feature flag. For free orgs, you will need to first contact Okta support (`support@okta.com`) to request email customization.
+However, when you disable the Enhanced Email Macros feature, your old customized templates will return. If you choose to disable the feature, you will lose access to the customizations done while the EA feature was enabled.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md:45]:
```suggestion
However, when you disable the Enhanced Email Macros feature, your old customized templates return. If you choose to disable the feature, you lose access to the customizations done while the EA feature was enabled.
```

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Review by albertchen-okta — APPROVED** [review]
lgtm thanks Van

**Pull request merged**